### PR TITLE
Allow usage without baseUrl

### LINKED ISSUE #52: baseUrl should not be required
Error:

```
Resolve error: Non-relative paths are not allowed when 'baseUrl' is not set. Did you forget a leading './'?
```

It’s perfectly valid to use `paths` in TypeScript without setting `baseUrl`, see https://github.com/microsoft/TypeScript/pull/40101.

### PR DESCRIPTION
**Pull request by alex-kinokon**
Fixes #52.

### CODE REVIEW

File: tests/specs/create-paths-matcher.ts [hunk 1]
```diff
@@ -31,15 +31,25 @@ export default testSuite(({ describe }) => {
 					'tsconfig.json': createTsconfigJson({
 						compilerOptions: {
 							paths: {
-								'@': ['src'],
+								'@/*': ['./src/*'],
```
Comment (privatenumber) [inline tests/specs/create-paths-matcher.ts:34]: Please revert the configuration change. We want to handle the case where users pass in a path without `./`.

**Comment by privatenumber** [comment]
I updated the test to demonstrate that tsc throws an error when `./` is not passed in:
https://github.com/privatenumber/get-tsconfig/commit/2119b1c56f788031705a6104f506670192fae897

**Comment by alex-kinokon** [comment]
tsc is still throwing because the `paths` is using `src` and not `./src`. I don’t think it has anything to do with the lack of `baseUrl`.

File: tests/specs/create-paths-matcher.ts (hunk 1 above)
Comment (alex-kinokon) [reply tests/specs/create-paths-matcher.ts:34]: To be clear, `baseUrl` can only be absent if `paths` is relative.

**Comment by privatenumber** [comment]
In the commit I added, notice Im asserting that the same error message about baseUrl missing is used in tsc's error output.

File: tests/specs/create-paths-matcher.ts (hunk 1 above)
Comment (privatenumber) [reply tests/specs/create-paths-matcher.ts:34]: You can add another test for that. Changing this one reduces coverage.

**Comment by alex-kinokon** [comment]
I don’t think the issue was valid. I closed it for now.

**Pull request closed without merging**